build.gradle

```diff
@@ -168,6 +168,14 @@ if (false) {
 
 
 subprojects {
+
+  // enable running :dependencies task recursively on all subprojects
+  // eg: ./gradlew allDeps
+  task allDeps(type: DependencyReportTask) {}
+  // enable running :dependencyInsight task recursively on all subprojects
+  // eg: ./gradlew allDepInsight --configuration runtime --dependency com.fasterxml.jackson.core:jackson-databind
+  task allDepInsight(type: DependencyInsightReportTask) doLast {}
+
```

Comment:
Can we submit this change to Apache Kafka?

Reply:
Sure, I can put it put it up there. Should I remove it from this one?

Reply:
Removed from this PR. I'll put up a separate PR against upstream.

Reply:
apache/kafka#7694